bazel/cargo/wasmtime/remote/BUILD.errno-0.2.8.bazel

```diff
@@ -55,7 +55,6 @@ rust_library(
     ] + selects.with_or({
         # cfg(unix)
         (
-            "@rules_rust//rust/platform:wasm32-wasi",
```

Comment:
This shouldn't be removed (the conditions for duplicated `select` deps were supposed to be combined, not removed). Could you restore this line?